### PR TITLE
Update it-IT translations

### PR DESCRIPTION
**Pull request by andsala**
* Updated translation keys for:
  * ColorPicker
  * FlatColorPicker
  * Dropdowns
  * Filter
  * Textbox
  * Label
  * Calendar
* Updated it-IT translations

### CODE REVIEW

File: messages/textbox/textbox.zh-TW.yml [hunk 1]
```diff
@@ -2,3 +2,6 @@ kendo:
   textbox:
     # The title of the clear button
     clearTitle: Clear
+
```
Comment (tsvetomir) [inline messages/textbox/textbox.zh-TW.yml:5]: @dtopalov `clearTitle` doesn't seem to be used. We can remove it as a separate fix.

**Review by tsvetomir — APPROVED** [review]
Thank you very much for your contribution! LGTM

**Review by dtopalov — APPROVED** [review]

**Comment by tsvetomir** [comment]
Merged manually in `develop`. Thank you so much @andsala! 🍻

**Comment by tsvetomir** [comment]
Released in [v1.35.0](https://github.com/telerik/kendo-angular-messages/releases/tag/v1.35.0)

**Pull request closed without merging**